omas/machine_mappings/d3d.py

```diff
@@ -1024,6 +1028,8 @@ def langmuir_probes_data(ods, pulse, _get_measurements=True):
         if data_present[i]:
             try:
                 r = mdsvalue('d3d', pulse=pulse, treename='langmuir', TDI=r'\langmuir::top.probe_{:03d}.r'.format(i)).data()
+                if r is None:
+                    raise ValueError()
```

Comment:
Why does MDS+ return `None` for the test. This causes a comparison of `None > 0` later. I put in this hotfix, but I am not happy with it because I don't understand how the old code worked. Also why does the test case have no data? That seems a bit pointless.